lib/active_ldap/ldif.rb

```diff
@@ -138,7 +138,7 @@ def read_base64_value
       end
 
       def read_external_file
-        uri_string = @scanner.scan(URI::ABS_URI)
+        uri_string = @scanner.scan(URI::ABS_URI).chomp
```

Comment:
BTW this part https://github.com/activeldap/activeldap/blob/master/lib/active_ldap/ldif.rb#L145 is probably also not needed, because uri_string should match URI regexp before https://github.com/activeldap/activeldap/blob/master/lib/active_ldap/ldif.rb#L141